.github/wiki/Home.md

```diff
@@ -4,56 +4,48 @@ This repository generates a new project based on our preferences, by running a s
 
-**Essential**:
+**Approved**:
 - [Timber](https://github.com/JakeWharton/timber): A logging library which is always used in all the projects we've worked on.
 - [Firebase App Distribution](https://firebase.google.com/docs/app-distribution): Continuous delivery always requires a boilerplate setup.
 - [Detekt](https://github.com/detekt/detekt): Code smell analysis is important to maintain code quality.
 - [Hilt](https://developer.android.com/training/dependency-injection/hilt-android): Dependency injection provides a good developer experience.
 - [Navigation Component](https://developer.android.com/guide/navigation/navigation-getting-started): A suite of libraries for in-app navigation which is recommended by Google.
 
-**Optional**:
-- [RxPermissions](https://github.com/tbruyelle/RxPermissions): Permissions powered by RxJava are regularly used, but not every project needs it.
-- [Bitrise](https://www.bitrise.io/): A CI/CD provider which is regularly used, but [GitHub Actions](https://github.com/features/actions) is our default choice.
-
 **Rejected**:
 - [Skeleton Layout](https://github.com/Faltenreich/SkeletonLayout): A progress indicator with visual feedback, it has a specific use case that is barely used.
 - [Mockito](https://github.com/mockito/mockito): A mocking framework which is barely used, because [Mockk](https://mockk.io/) is our default choice.
 
-Keep in mind, the features are based on _our team's_ requirements. In case the client has different requirements or requests, we can consider adding them as optional features if they occur regularly.
+Keep in mind, the features are based on _our team's_ requirements. In case the client has different requirements or requests, we can consider adding them as features if they occur regularly.
 
 Please note that the above examples are not definitive as new and existing libraries keep on emerging and evolving.
 
 Before an issue can be worked on, it must go through our voting process. 
 
 How do we vote on an issue? 🗳
 
-- It is the responsibility of the RFC creator to label their proposed change as **essential** or **optional**.
-- If we agree with the RFC, we must react with 👍. 
-  - If there are 3 x 👍, then the issue is approved.
-- If we disagree with the RFC, we must react with 👎 and leave a comment explaining why. 
-  - If there are 3 x 👎, then the issue is rejected.
+- If we agree with the proposed issue, we must react with 👍. 
+- If we disagree with the proposed issue, we must react with 👎 and leave a comment explaining why. 
 - If there are differing opinions, then the repository maintainer must resolve it.
 
+When is an issue approved/rejected? 🕐 
+
+  - If at least half of the team members react with 👍, then the issue is approved and the `status : approved` label is added.
+  - If at least half of the team members react with 👎, then the issue is rejected and the `status : rejected` label is added.
+  - If the vote is tied, then the repository maintainer must resolve it.
```

Comment:
```suggestion
  - If there are at least 3 of the team members react with 👍, then the issue is approved and the `status : approved` label is added.
  - If  If there are at least 3 of the team members react with 👎, then the issue is rejected and the `status : rejected` label is added.
  - If the vote is tied, then the repository maintainer must resolve it.
  
To facilitate a discussion and reach a decision on a solution, our team follows these steps:
  1. Initiate a discussion using the RFC template.
  2. A solution will be considered approved if any of its options receive at least 50% of the team member votes.
```

What do you think?

Reply:
@lydiasama I like the second part but it makes things a little unclear IMO, it's like there are two choices:
1. Open an issue that only needs 3 votes
2. Open an RFC that needs at least 7 (or 50%) votes

Should we specify which one we should open if we want to suggest a change to the template?
Or would issues be for things like chores and bugs while RFC discussions are for features?

CC: @Tuubz @luongvo

Reply:
@ryan-conway Good question. Currently, we do both.

Reply:
@ryan-conway Maybe it's time to create a RFC discussion for this? 😄

Reply:
@lydiasama I dont quite catch your ideas too 🤔 The first part I think we should have at least 50% of vote instead of just 3 votes, so then it will be consistent for all the types of issue 🤔 

Would you mind explaining it a little bit more why we should have 2 approaches when finalizing an RFC discussion 😄 That would be great

Reply:
@thiennguyen0196 For example, when someone finds a bug, if we only have the RFC poll with 50% of the member vote, the process to resolve a bug will be quite cumbersome. Instead, if we create simple issues when there are improvement chores of bugs, we can get just 3 approval and start fixing/implementing them.

Reply:
Got it @lydiasama It is clear now 👍 Thank you for your explanation